### PR TITLE
Fix libvirt_manager and reproducer roles

### PR DESCRIPTION
**Pull request by cjeanner**
With all of the changes happening in order to rework the layout, add
support for devscripts and related tasks, it appears the reproducer
broke. More specifically, the network part was out, due to hard-coded
values that leaked following some hackathon (really, hack-a-ton) session
that happened lately.

This patch puts things square again and allows to deploy the needed
bits.

In order to ensure things are working properly, we also do some cleaning
in molecule:
- removing existing tests, since they are mostly the same and are
  consuming install_yamls, with dry_run
- adding a new test that will actually deploy a bunch of VM, consuming
  most of the features available in the role (layout, multi-nic,
  fixed_ips)

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [X] Appropriate documentation exists and/or is up-to-date:
  - [X] README in the role

### CODE REVIEW

**Comment by cjeanner** [comment]
I want to extend the molecule tests of the libvirt_manager to cover the deploy_layout part.

**Review by psathyan — COMMENTED** [review]
/lgtm

File: ci_framework/roles/libvirt_manager/README.md [hunk 1]
```diff
@@ -27,3 +27,18 @@ Used for checking if:
...
+## Parameters imported from the reproducer role
+The following parameters are usually set in the [reproducer](./reproducer.md) context.
+The parameters listed here are therefore merely proxies to the ones set in the reproducer,
+and have the same name, less the role prefix. Default values are the same as the
+reproducer role.
+
+* `cifmw_libvirt_manager_private_nic`: `{{ cifmw_reproducer_private_nic | default('eth1') }}`
+* `cifmw_libvirt_manager_ctl_ip4`: `{{ cifmw_reproducer_ctl_ip4 | default('192.168.122.11') }}`
+* `cifmw_libvirt_manager_ctl_gw4`: `{{ cifmw_reproducer_ctl_gw4 | default('192.168.122.1') }}`
+* `+cifmw_libvirt_manager_crc_ip4`: `{{ cifmw_reproducer_crc_ip4 | default('192.168.122.10') }}`
```
[the remark this answers is not in the document]
Comment (cjeanner) [reply ci_framework/roles/libvirt_manager/README.md]: woops, good catch

**Review by psathyan — APPROVED** [review]
/lgtm

**Review by raukadah — APPROVED** [review]
/approve

**Pull request merged**